orc8r/cloud/go/services/orchestrator/obsidian/models/defaults.go

```diff
@@ -57,6 +63,7 @@ func NewDefaultNetwork(networkID string, name string, description string) *Netwo
 		DNS:          NewDefaultDNSConfig(),
 		Features:     NewDefaultFeaturesConfig(),
 		SentryConfig: NewDefaultSentryConfig(),
+		StateConfig:  NewDefaultStateConfig(),
```

Comment:
is there already a unit test for this func? If not, we can add a simple one here too

Reply:
i think these are just helper functions for unit tests, looking at code usages i don't see it used anywhere outside of tests